sale_automatic_workflow/__init__.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
 # © 2011 Akretion Sébastien BEAU <sebastien.beau@akretion.com>
```

Comment:
replace `©` sign w/ `Copyright` :)

Reply:
second opinion as well please, I didn't write it, and there is nothing saying it isn't OK.  In all there are 27 of them.